src/components/MyPage.js

```diff
@@ -110,7 +110,8 @@ class MyPage extends Component {
   signOut() {
     try {
       UserUtil.signOut((result, error) => {
-        AsyncStorage.removeItem('token', () => { Actions.login(); });
+        let keys = ['token', 'filter'];
```

Comment:
yes, actually we didn't save user's filter data.
So I saved the filter data on local. Let's talk about this issue @KyuaKwon @Stacyyya